src/app/index.html

```diff
@@ -49,19 +49,76 @@
   </stache-image>
 
   <stache-page-anchor>
-    Videos
-  </stache-page-anchor>
-
-  <stache-video
-    videoSource="https://player.vimeo.com/video/121445495?title=0&portrait=0&byline=0">
-  </stache-video>
-
-  <stache-page-anchor>
     Include
   </stache-page-anchor>
 
   <stache-include
     fileName="example.html">
   </stache-include>
 
+  <stache-tutorial>
```

Comment:
Mind if we clean-up and "professionalize" the Home page tutorial example and delete the Learn page stuff?

**Also, it'd be nice to see:**
1. An example of a step that only includes a header
2. An example of a step that only includes body content
3. An example of a SKY Alert between two steps